sdcm/cluster.py

```diff
@@ -453,23 +455,33 @@ def install_scylla_bench(self):
         self.is_scylla_bench_installed = True
 
     @retrying(n=3)
-    def install_cassandra_harry(self):
+    def install_cassandra_harry(self, cassandra_harry_folder_path: str = ""):
+        cassandra_harry_path = \
+            Path(cassandra_harry_folder_path if cassandra_harry_folder_path else realpath("")) / "cassandra-harry"
+        pom_xml_path = cassandra_harry_path / "pom.xml"
         if self.distro.is_rhel_like:
             self.remoter.sudo("yum install -y git make maven")
         else:
             self.remoter.sudo(shell_script_cmd("""\
                 apt-get update
                 apt-get install -y git make maven
             """))
-        self.remoter.run(shell_script_cmd("""\
-            rm -rf cassandra-harry
-            git clone https://github.com/apache/cassandra-harry
-            cd cassandra-harry
-            git checkout 26bb6696ba1b18ff5c062d780a9360e750691052
+        self.remoter.run(shell_script_cmd(f"""\
+            rm -rf {cassandra_harry_path}
+            git clone https://github.com/apache/cassandra-harry {cassandra_harry_path}
+        """))
+        with pom_xml_path.open(mode="r", encoding="utf-8") as file:
+            # This line should be deleted because "make mvn" failed because the
+            # "org.apache.cassandra:cassandra-dtest-shaded:jar:4.1-58515c2de6" dependency is missing
+            pom_data_lines = [line for line in file.readlines() if "<module>harry-integration</module>" not in line]
+        with pom_xml_path.open(mode="w", encoding="utf-8") as file:
+            file.writelines(pom_data_lines)
+        self.remoter.run(shell_script_cmd(f"""\
+            cd {cassandra_harry_path}
             make mvn
         """))
-        result = self.remoter.run('realpath ~/cassandra-harry/scripts/cassandra-harry')
-        self.remoter.sudo(f"ln -s {result.stdout.strip()} /usr/bin/cassandra-harry", ignore_status=True)
+        # To overcome this "ln: failed to create symbolic link '/usr/bin/cassandra-harry': File exists" error.
+        self.remoter.sudo(f"ln -sf {cassandra_harry_path} /usr/bin/cassandra-harry", ignore_status=True)
```

Comment:
shouldn't you check first if `/usr/bin/cassandra-harry` before you try to create this link?